homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/gravatar/components/GravatarHeader.kt

```diff
@@ -0,0 +1,101 @@
+package com.gravatar.app.homeUi.presentation.home.gravatar.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.systemBarsPadding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.BlurredEdgeTreatment
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.gravatar.app.homeUi.R
+import com.gravatar.restapi.models.Avatar
+import com.gravatar.ui.components.ComponentState
+import java.net.URI
+import com.gravatar.ui.components.atomic.Avatar as GravatarAvatar
+
+@Composable
+fun GravatarHeader(
+    avatar: Avatar?,
```

Comment:
One more thing, more as an FYI: This will work on the GravatarTab, but on the others we might have to use the hash to get the current avatar, so maybe it's worth taking this into consideration already.